### PR TITLE
feat(suite): add promo banner for wallet remember into receive form

### LINKED ISSUE #12403: Receive page with disconnected trezor (view-only)
- banner
- show button for unverified address

Screenshot 2024-05-15 at 12 11 10

https://www.figma.com/design/wGshbwWuR7shAA1i1UBqR8/branch/1O5PSi8KzMpeVyFMrEMZeA/Remember-Mode?node-id=2354-40396&t=03r75bJYrFjp9AqS-0

### PR DESCRIPTION
**Pull request by peter-sanderson**
image

image

resolves #12403

### CODE REVIEW

File: packages/components/src/components/Warning/Warning.tsx [hunk 1]
```diff
@@ -103,7 +103,6 @@ const Wrapper = styled.div<WrapperParams>`
     display: flex;
     ${typography.hint}
     gap: ${spacingsPx.sm};
-    justify-content: ${({ $withIcon }) => !$withIcon && 'center'};
```
Comment (peter-sanderson) [inline packages/components/src/components/Warning/Warning.tsx:106]: I really dont like that `withIcon` changes the alignment. It shall be independent

Comment (peter-sanderson) [reply packages/components/src/components/Warning/Warning.tsx:106]: (Alignment from the icon presence)

File: packages/suite/src/views/wallet/receive/components/ConnectDevicePromo.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    const selectedDevice = useSelector(selectDevice);
+    const selectedDeviceModelInternal =
+        selectedDevice?.features?.internal_model || DEFAULT_FLAGSHIP_MODEL;
+
+    return (
+        <Warning variant="warning" withIcon={false}>
+            <Flex direction="row" alignItems="stretch">
+                <Flex direction="column" alignItems="start">
+                    <Text typographyStyle="highlight" variant="warning">
+                        Receive address can’t be verified
```
Comment (peter-sanderson) [inline packages/suite/src/views/wallet/receive/components/ConnectDevicePromo.tsx]: TODO: TX_messages

**Review by peter-sanderson — COMMENTED** [review]
:heavy_check_mark:

**Review by jvaclavik — APPROVED** [review]

**Pull request merged**